src/model/ZoneHVACTerminalUnitVariableRefrigerantFlow_Impl.hpp

```diff
@@ -123,7 +126,7 @@ namespace model {
 
       Schedule supplyAirFanOperatingModeSchedule() const;
 
-      HVACComponent supplyAirFan() const;
+      boost::optional<HVACComponent> supplyAirFan() const;
```

Comment:
I'm changing the Impl side of the API to assume an optional fan, as it should be now.